index/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Untitled Document</title>
+    <link rel="stylesheet" href="C:\Users\Admin\Desktop\Kodilla\10.2_jQuery\css\style.css" type="text/css">
```

Comment:
this styles will work only on your computer, path should look `css/style.css` or `./css/style.css`